src/Core/Geographic/Coordinates.js

```diff
@@ -18,6 +19,9 @@ export function ellipsoidSizes() {
 
 const ellipsoid = new Ellipsoid(ellipsoidSizes());
 
+proj4.defs('EPSG:3946',
```

Comment:
Suggestion:
- Remove EPSG:3946 from Coordinates.js
- Add a function "defineCrs(name, unit, projString)"
- Call this function in index.html